src/LiveComponent/CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # CHANGELOG
 
+## 2.29.0
+
+-   Added new modifier `limit` and new Hooks for Poll System.
+    - New Hooks: `poll:started`, `poll:running`, `poll:paused`, `poll:stopped`, `poll:error`
+    - Handling Count loop and pause, start, stop, resume, error events
+
 ## 2.28.0
 
--   Add new modifiers for input validations, useful to prevent unnecessary HTTP requests:
+-   Add new modifiers for input validations, useful to prevent uneccessary HTTP requests:
```

Comment:
```suggestion
-   Add new modifiers for input validations, useful to prevent unneccessary HTTP requests:
```

It seems `unneccessary` contains two N: https://en.wiktionary.org/wiki/unnecessary